packages/editor-ui/src/Interface.ts

```diff
@@ -577,7 +577,6 @@ export interface IRootState {
 	pushConnectionActive: boolean;
 	saveDataErrorExecution: string;
 	saveDataSuccessExecution: string;
-	saveManualExecutions: boolean;
```

Comment:
turns out the value of this is wrong and we are not even using it in workflow settings...